euphorichubbbb.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "@euphorichubbbb"
+project: "Spectrum analyzer headphone"- Suraga Devraj
```

Comment:
```suggestion
project: "Spectrum analyzer headphone"
```

Reply:
we're using this section "metadata" for each project so we'd need to keep it standard with just the project name hehe, thank you!

Reply:
okay ill correct that..